server/deletePartners.js

```diff
@@ -0,0 +1,5 @@
+import models from './models/index';
+
+models.sequelize.query('DELETE FROM "partners"').then(([, metadata]) => {
```

Comment:
@nzediegwu1, Shall this be able to remove all the records or only the null values.

Reply:
All, since the error thrown originated from a migration which adds location column to partner table, and as such, no current record has location field